torch/_dynamo/decorators.py

```diff
@@ -437,19 +437,28 @@ class directly; instead, use :func:`mark_dynamic`.
 
 
 @forbid_in_graph
-def mark_unbacked(t, index):
+def mark_unbacked(t, index, strict=False):
```

Comment:
do you want to make strict=True default in a later diff? or do you want to make mark_dynamic(strict = False) instead?